manifests/init.pp

```diff
@@ -27,18 +27,32 @@
 #
 # @param manage_properties
 #   Boolean specifying whether or not to manage the catalina.properties file. Defaults to true.
+#
+# [*manage_dirs*]
+#   Boolean specifying whether or not to manage sub-directories under $catalina_base.
+#   Defaults to true.
+#
+# [*dir_list*]
+#   Array of sub-directories to manage under $catalina_base.
+#   Defaults to tomcat::params::dir_list.
+#
+# [*dir_mode*]
+#   Mode to use for managed sub-directories under $catalina_base. Defaults to '2770'.
 class tomcat (
   $catalina_home       = $::tomcat::params::catalina_home,
   $user                = $::tomcat::params::user,
   $group               = $::tomcat::params::group,
+  Array $dir_list            = $::tomcat::params::dir_list,
+  String $dir_mode           = $::tomcat::params::dir_mode,
```

Comment:
I'd rather not add more parameters to the base class; see https://github.com/puppetlabs/puppetlabs-tomcat/pull/145 for a detailed reasoning.